features/aanschrijfwijze.feature

```diff
@@ -2,17 +2,17 @@
 
 # User story #13
 Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze van mijn burgers
-  Attribuut aanschrijfwijze bij een persoon wordt gevuld door de provider om op deze wijze op eenduidige wijze een persoon te kunnen aanschrijven.
-  Bij het samenstellen van de aanschrijfwijze worden academische titels vooralsnog niet opgenomen. Academische titels zijn geen authentiek gegeven en daarom buiten scope geplaatst.
+  De attributen aanschrijfwijze.naam en aanschrijfwijze.aanspreekvorm bij een persoon worden gevuld door de provider om op deze wijze op eenduidige wijze een persoon te kunnen aanschrijven.
+  Bij het samenstellen van de aanschrijfwijze.naam worden academische titels vooralsnog niet opgenomen. Academische titels zijn geen authentiek gegeven en daarom buiten scope geplaatst.
   
-  De aanschrijfwijze wordt gebruikt als eerste regel in de adressering op een envelop, of links bovenaan een brief, direct boven het adres.
+  De aanschrijfwijze.naam wordt gebruikt als eerste regel in de adressering op een envelop, of links bovenaan een brief, direct boven het adres.
```

Comment:
de toevoeging .naam maakt het niet duidelijk. je zegt dat aanschrijfwijze.naam de eerste regel is. Maar de aanspreekvorm (voorheen: regelVoorafgaandAanAanschrijfwijze) wordt daarvoor geplaatst.

De aanschrijfwijze wordt gebruikt als eerste regel(s) in de adressering op een envelop

Reply:
Wijzigen van 'regel' in 'regel(s)' maakt mijn aanpassing idd onnodig.

Reply:
Aangepast